docs/units/standard_units.rst

```diff
@@ -48,6 +48,78 @@ all the existing predefined units of a given type::
     u            | 1.66054e-27 kg  | Da, Dalton                       ,
   ]
 
+
+Prefixes
+--------
+
+Most units can be used with prefixes, with both the standard SI prefixes and
+the IEEE 1514 binary prefixes (for ``bit`` and ``byte``) supported:
+
++------------------------------+
+|  Available decimal prefixes  |
++--------+-------------+-------+
+| Symbol |    Prefix   | Value |
```

Comment:
I think these column names are potentially confusing because "Symbol" needs to be used with astropy units, not the "Prefix" (as stated).  Perhaps these columns should be renamed "Prefix", "Name", "Value"?  Same for the binary prefixes.